editor/selectors.js

```diff
@@ -3,6 +3,7 @@
  */
 import moment from 'moment';
 import { first, last, get } from 'lodash';
+import { createSelector } from 'reselect';
```

Comment:
@youknowriad That's incorrect usage. There should be two arguments at most, so the second should be changed to:

```js
( state ) => [
	state.editor.blockOrder,
	state.editor.blocksByUid,
]
```